relion/protocols/protocol_subtract.py

```diff
@@ -171,14 +311,20 @@ def createOutputStep(self):
 
         self.reader = convert.createReader(alignType=ALIGN_PROJ)
         tableName = 'particles@' if self.IS_GT30() else ''
+        if self.isRelionInput:
```

Comment:
not accepted. Please do not modify working code or open an issue if there's a problem

Reply:
!? I do not understand the problem here.  I do not want to modify the code you wrote, but the way the angles and shifts are assigned is not valid for the option I am implementing.  I do not understand what I am supposed to do here.